### PR TITLE
Added Map Marker Support to Core Atlas

### PR DESCRIPTION
**Pull request by felangel**

### CODE REVIEW

File: packages/atlas/lib/src/coordinate_pair.dart [hunk 1]
```diff
@@ -1,6 +1,14 @@
 import 'package:meta/meta.dart';
 
+/// A pair of latitude and longitude coordinates.
+/// The `latitude` and `longitude` are stored as degrees.
 class CoordinatePair {
```
Comment (jorgecoca) [inline packages/atlas/lib/src/coordinate_pair.dart]: Why not just `Coordinate`?

File: packages/atlas/lib/src/marker.dart [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import 'package:meta/meta.dart';
+import 'package:atlas/atlas.dart';
+
+/// Marks a geographical location on the map.
+class Marker {
+  /// Uniquely identifies a `Marker`.
+  final String id;
+
+  /// The location where the `Marker` is drawn is represented as a `CoordinatePair`.
+  final CoordinatePair position;
```
Comment (jorgecoca) [inline packages/atlas/lib/src/marker.dart]: I would keep the variable named as the class.. `final Coordinate coordinate`

**Review by jorgecoca — CHANGES_REQUESTED** [review]

**Comment by felangel** [comment]
@jorgecoca please re-review

File: packages/atlas/README.md [hunk 3]
```diff
@@ -35,7 +36,7 @@ class MyApp extends StatelessWidget {
 
 class AtlasSample extends StatelessWidget {
   final CameraPosition _initialCameraPosition = CameraPosition(
```
Comment (tchabot22) [inline packages/atlas/README.md:38]: Unrelated to MapMarkers but I think we should discuss what we think we need for CameraPosition.  I think target and zoom are sufficient for now.

**Review by tchabot22 — APPROVED** [review]

**Review by noahmateen — APPROVED** [review]

File: packages/atlas/lib/src/marker.dart (hunk 2 above)
Comment (felangel) [reply packages/atlas/lib/src/marker.dart]: Renamed to `LatLng` 👍

File: packages/atlas/lib/src/coordinate_pair.dart (hunk 1 above)
Comment (felangel) [reply packages/atlas/lib/src/coordinate_pair.dart]: renamed to `LatLng` 👍

**Review by jorgecoca — APPROVED** [review]

**Pull request merged**